### PR TITLE
Awesome Books: Implement Awesome Books App Using ES6 and Luxon

### PR DESCRIPTION
**Pull request by rmiyoyo**
# Work Done
- I converted the webapp done last week in the collaboration project into the use of ES6 modules
- Used Luxon to implement the time feature.
- Added transitions and animations to the project.
- Ran and fixed linter errors.

### CODE REVIEW

**Review by Yerimah — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️ 

Hi @rmiyoyo, 👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

212489643-ad7fc55f-bde0-4662-8461-10afd54ca2d0


## Highlights
- No linter errors 👍 
- Descriptive PR summary and title 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rmiyoyo** [comment]
# Work Done After Review
- Used arrow syntax for functions in the mobile.js file and added it to modules for future use.

File: mobile.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+function saveFormDataToLocalStorage() {
+  const nameInput = document.querySelector('input[name="name"]');
+  const emailInput = document.querySelector('.email-box');
```
Comment (Zuhaib042) [inline mobile.js:3]: - [ ] I can see that you have created another mobile.js with the code of this file and using ES6 arrow syntax but the problem is that this file still exists here, what he meant was to take this file into modules and then suggested to export it in index.js.
- [ ] I will recommend you to remove this file from here, as you taking the contents of this file in the modules directory. having the same content in two separate files is against DRY principle so please consider fixing it.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @rmiyoyo 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
No linter errors 👍
Descriptive PR summary and title 👍

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: mobile.js (hunk 1 above)
Comment (rmiyoyo) [reply mobile.js]: Hi @Zuhaib042 Sorry I removed the file but I did not commit the change.

**Comment by rmiyoyo** [comment]
# Work Done After Second Review
- Removed mobile.js file from the root folder

**Review by algerina — APPROVED** [review]
Hi @rmiyoyo ,

Great job so far! 

May I highlight the following: 
- [x] Your project is working fine👏  ✔️ 
- [x] You set up all the requirements requested by the previous TSE✔️ 



## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**